packages/db/drizzle/0020_sync_request_type_enum.sql

```
@@ -0,0 +1,6 @@
+ALTER TABLE "update_requests" ALTER COLUMN "request_type" SET DATA TYPE text;--> statement-breakpoint
+UPDATE "update_requests" SET "request_type" = 'create_ao_and_location_and_event' WHERE "request_type" = 'create_location';--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)